libdirections-offboard/src/test/java/com/mapbox/navigation/route/offboard/MapboxOffboardRouterGenerationTest.kt

```diff
@@ -10,7 +11,7 @@ class MapboxOffboardRouterGenerationTest {
 
     @Before
     fun setUp() {
-        offboardRouter = MapboxOffboardRouter()
+        offboardRouter = MapboxOffboardRouter(mockk(), "pk_token")
```

Comment:
This test should be renamed to `MapboxOffboardRouterTest.kt` and actually test the module.

Reply:
I see this is happening here https://github.com/mapbox/mapbox-navigation-android/pull/2270 :+1: